packages/commands/src/store/reducer.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * WordPress dependencies
+ */
+import { combineReducers } from '@wordpress/data';
+
+/**
+ * Reducer returning the registered commands
+ *
+ * @param {Object} state  Current state.
+ * @param {Object} action Dispatched action.
+ *
+ * @return {Object} Updated state.
+ */
+function commands( state = {}, action ) {
```

Comment:
Correct me if I'm wrong: we aren't using this reducer, right?

Reply:
You mean right now, we don't have any "static" commands. Yes. But It's supported by the component and I used to have in the initial state of the PR.

Reply:
yes, it's the same one.